src/frontend/src/lib/components/ui/ExternalLink.svelte

```diff
@@ -9,6 +12,15 @@
 	export let color: 'blue' | 'inherit' = 'inherit';
 	export let fullWidth = false;
 	export let styleClass = '';
+	export let trackEventParams: TrackEventParams | undefined = undefined;
```

Comment:
nitpick: I would call the property `trackEvent` and rename the function `trackEvent as trackEventServices } from ...` to avoid conflicts. That way consuming the component is a bit less verbose.

Reply:
done!